config/options.go

```diff
@@ -252,7 +274,7 @@ func NewDefaultOptions() *Options {
 func NewOptionsFromConfig(configFile string) (*Options, error) {
 	o, err := optionsFromViper(configFile)
 	if err != nil {
-		return nil, fmt.Errorf("config: options from viper %w", err)
+		return nil, fmt.Errorf("config: options from config file %w", err)
```

Comment:
@travisgroth fyi, snuck this change in. It's very much a me problem but when I'd see this in the logs I would assume the error originated from viper, and not us.

Reply:
👍